mobilizon_reshare/config/config.py

```diff
@@ -20,6 +20,9 @@
     # url of the main Mobilizon instance to download events from
     Validator("source.mobilizon.url", must_exist=True, is_type_of=str),
     Validator("source.mobilizon.group", must_exist=True, is_type_of=str),
+    Validator("db_path", must_exist=True, is_type_of=str),
+    Validator("locale", must_exist=True, is_type_of=str, default="en-us"),
+    Validator("log_dir", must_exist=True, is_type_of=str),
```

Comment:
This is not mandatory and nothing checks for a `'log_dir'` key in settings. In our example file we use it just because it's easier to figure out but the actual conf is the logging magic -> https://github.com/Tech-Workers-Coalition-Italia/mobilizon-reshare/blob/master/mobilizon_reshare/settings.toml#L37

Reply:
so it should be removed completely?

Reply:
yes i believe so